pages/cart.js

```diff
@@ -0,0 +1,34 @@
+const { I } = inject();
+
+module.exports = {
+  emailInput: { css: '#email' },
+  passwordInput: { css: '#passwd' },
+  submitLogin: { css: '#SubmitLogin' },
+  agreement: { css: '#cgv' },
+  productPrice: { css: '#our_price_display' },
+  finalPrice: { css: '#total_price' },
+  hookPayment: {xpath: '//*[@id="HOOK_PAYMENT"]/div[1]/div/p/a'},
+
+
+  async proceedToCart() {
```

Comment:
Name of function does not correspond its functionality.
You complete purchase here, not just proceed to cart.